libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -263,6 +263,30 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     ],
     keywords: ["avax", "avalanche"],
   },
+  avalanchepchain: {
+    type: "CryptoCurrency",
+    id: "avalanchepchain",
+    coinType: 9000,
+    name: "Avalanche (P-Chain)",
+    managerAppName: "Avalanche",
+    ticker: "AVAXP",
```

Comment:
@gre suggested [here](https://github.com/LedgerHQ/ledger-live/pull/1292#discussion_r995772626) it might be an issue if two currencies (in this case, p chain and c chain avax) have the same ticker. Is this not an issue anymore?

Reply:
Yup, they use the same currency. `AVAX` it is!